shopfloor_batch_automatic_creation/tests/test_batch_create.py

```diff
@@ -178,7 +178,7 @@ def test_volume(self):
 
     def test_cluster_picking_select(self):
         self.menu.sudo().batch_create = True
-        self.menu.sudo().batch_create_max_pickings = 2
+        self.menu.sudo().batch_create_max_picking = 2
```

Comment:
seems legit, but: does it have any effect on tests? Semes not if it was passing before :thinking:

Reply:
Thanks @simahawk , we are still scratching our head as we don't understand why the test was passing before AND the test was failed recently.